pkg/sctp/examples/client/client.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io"
+	"strconv"
+	"time"
+
+	"log"
+
+	petname "github.com/dustinkirkland/golang-petname"
```

Comment:
Is this the same library we're using in Helmit? We should just make sure we're using the same library for this so we're not adding unnecessary dependencies.

Reply:
Yeah it is the same library.